src/CLITests/CLITests.csproj

```diff
@@ -2,6 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>netcoreapp3.1</TargetFramework>
+    <RootNamespace>AxeWindowsCLITests</RootNamespace>
```

Comment:
Should this be `Axe.Windows.CLITests` for consistency with the rest?

Reply:
The CLI namespace is AxeWindowsCLI, so this is AxeWindowCLITests.